task_assign/rt[] rt_vect